search/Search.tcl

```diff
@@ -158,21 +159,31 @@ proc find_timing_paths_cmd { cmd args_var } {
 
   set corner [parse_corner_or_all keys]
 
-  set endpoint_count 1
-  if [info exists keys(-endpoint_count)] {
-    set endpoint_count $keys(-endpoint_count)
-    if { $endpoint_count < 1 } {
-      sta_error 512 "-endpoint_count must be a positive integer."
-    }
+  set endpoint_path_count 1
+  if { [info exists keys(-endpoint_count)] } {
+    # deprecated 2024-11-22
+    sta_warn 502 "find_timing_paths -endpoint_count is deprecated. Use -endpoint_path_count instead."
```

Comment:
The warnings should be for the command name $cmd, not the internal function.

Reply:
Sorry, will fix